test/e2e/kubevirt.go

```diff
@@ -1491,7 +1537,7 @@ runcmd:
 				topology: "layer2",
 				role:     "primary",
 			}),
-			Entry(nil, testData{
+			XEntry(nil, Label("TODO", "SDN-5490"), testData{
```

Comment:
Please mention in the commit sha why these entries are xfailed.

Reply:
Well we have the jira -> SDN-5490, it's all there.

Reply:
OK I guess that's enough.